src/OpenTelemetry/Trace/Sampler.cs

```diff
@@ -18,7 +18,7 @@
 namespace OpenTelemetry.Trace
 {
     /// <summary>
-    /// Sampler to select data to be exported. This sampler executes before Activity object is created.
+    /// Controls the number of samples of traces collected and sent to the backend.
```

Comment:
FYI, this is borrowed from the spec https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#sampling.